src/main/java/com/blamejared/crafttweaker/impl_native/tileentity/ExpandTileEntity.java

```diff
@@ -0,0 +1,12 @@
+package com.blamejared.crafttweaker.impl_native.tileentity;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.tileentity.TileEntity;
+
+@ZenRegister
+@Document("vanilla/api/tileentity/MCTileEntity")
+@NativeTypeRegistration(value = TileEntity.class, zenCodeName = "crafttweaker.api.tileentity.MCTileEntity")
+public class ExpandTileEntity {}
```

Comment:
Any members for this or is it just registered so that other classes that e.g. have it as parameters work?

Reply:
This is currently being registered only so that other classes that need it as a parameter can work. I'd like to also use this to expose some information, but I don't know what would be considered safe to expose and what wouldn't in this context, so I believe there needs to be some discussion